pkg/interactive/helper.go

```diff
@@ -50,7 +50,7 @@ func GetInstallerRoleArn(r *rosa.Runtime, cmd *cobra.Command,
 	role := aws.AccountRoles[aws.InstallerAccountRole]
 	roleARN := defaultInstallerRoleArn
 	// Find all installer roles in the current account using AWS resource tags
-	roleARNs, err := awsClient.FindRoleARNs(aws.InstallerAccountRole, minMinorVersion)
```

Comment:
@gdbranco I wasn't sure if it should be here only roles with hosted CP managed policies?

Reply:
I think it would require to add this arg to GetInstallerRoleArn instead and we can check what is using it better

I think that for now all uses will be false, but worth checking to be sure
I know that for OIDC Config it shouldn't matter

Reply:
Changed, thanks